test/reporter-urlparams.js

```diff
@@ -1,23 +1,26 @@
 if ( !location.search ) {
 	location.replace( "?implicit&explicit=yes&array=A&array=B&escaped%20name&" +
-		"module=urlParams&notrycatch&custom&customArray=a&customArray=b" );
+		"module=urlParams+module&notrycatch&custom&customArray=a&customArray=b&" +
+    "filter=urlParams%20module" );
```

Comment:
The spacing is off here. We allow up to 100 characters per line (where tab width is 4), so it might fit on one. But if not, please align it with the above.